statefun-e2e-tests/statefun-smoke-e2e-java/src/main/java/org/apache/flink/statefun/e2e/smoke/CommandInterpreterAppServer.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.statefun.e2e.smoke;
```

Comment:
It seems like a lot of the dependency complication come from the fact that all the classes of `statefun-smoke-e2e-driver`, `statefun-smoke-e2e-java`, and `statefun-smoke-e2e-embedded` all reside in the package `org.apache.flink.statefun.e2e.smoke`.

In reality, since we may be including more than one of these in the classpath at the same time, there are some class clashes that are not apparent right away.

Lets be safe here, and use different packages for the respective modules, e.g. `rg.apache.flink.statefun.e2e.smoke.<driver/java/embedded>`.

Reply:
Agree.